study-builder/studies/testboston/snippets/consent-section4.conf

```diff
@@ -95,6 +95,29 @@
       "blockType": "CONTENT",
       "shownExpr": """user.studies["testboston"].forms["CONSENT"].questions["WHO_FILLING"].answers.hasOption("SELF")"""
     },
+    {
+      "titleTemplate": null,
+      "bodyTemplate": {
+        "templateType": "HTML",
+        "templateText": """
+          <div>
+            <h3>First Name</h3>
+            <p>$DDP_PARTICIPANT_FIRST_NAME</p>
+          </div>
+          <div>
+            <h3>Last Name</h3>
+            <p>$DDP_PARTICIPANT_LAST_NAME</p>
+          </div>
+          <div>
+            <h3>Date</h3>
+            <p>$DDP_DASHED_DATE</p>
+          </div>
+        """,
+        "variables": []
```

Comment:
Here is an example of activity definition using those special vars. Note that no variable objects are added to the `variables` list for the special vars.